README.md

```diff
@@ -120,28 +58,18 @@ Here are possbile values (warn by default)
 - Put this generator closer to Fabric, or even on the same machine. This is to prevent network bandwidth from being the bottleneck.
 
 - Increase number of messages per block in your channel configuration may help
+- [Workflow](docs/workflow.md)
+
 
-## Help us improve
 
-If you wish for new features or encounter any bug, please feel free to open [issue](https://github.com/guoger/tape/issues), and we always welcome [pull request](https://github.com/guoger/tape/pulls).
+## [How to Contribute](CONTRIBUTING.md)
 
-If you are reporting an issue, please generously turn on debug log with `export TAPE_LOGLEVEL=debug` and paste log in the issue
+## [Maintainers](MAINTAINERS.md)
 
-## Development
-<details>
-<summary>Click to expand</summary>
 
-### Tape workflow
+## LICENSE
+Hyperledger Project source code files are made available under the Apache License, Version 2.0 (Apache-2.0), located in the [LICENSE](LICENSE) file.
 
-Tape consists of several workers that run in goroutines, so that the pipeline is highly concurrent and scalable. Workers are connected via buffered channels, so they can pass products around.
 
-![tape workflow](./docs/images/tape.jpeg)
-</details>
 
-## Maintainers
 
-| Name   | mail                     | github-ID               
-| ------ | ------------------------ | ----------- | 
-| Jay Gou | guojiannan1101@gmail.com | guoger      |
-| Sam Yuan   | yy19902439@126.com       | SamYuan1990 | 
-| Stone Cheng   | chengyang418@163.com     | stone-ch    | 
```

Comment:
not sure why we need to delete this section? IMO this makes readme more complete

Reply:
I planned to move this section to MAINTAINERS.md. 
How about we set the title ##Maintainers here as a link to MAINTAINERS.md ?